news/20200706.doc

```diff
@@ -0,0 +1 @@
+Fix docstring in exceptions.py
```

Comment:
Is this supposed to have a newline at the end or not?

Reply:
assert-news expect only one line per newsfile. If there is more than one line then it'll fail.

Reply:
It looks like there is a newline at the end of this file. Hopefully assert-news is OK with this, as vim usually ensures there is always a newline at the end of files when saving.

Reply:
Cool, looks like assert-news was OK with the newline at the end of the file. I'm glad. https://dev.azure.com/mbed-tools/mbed-build/_build/results?buildId=2791&view=logs&s=204402d8-1b03-5b49-8370-ed4d72352645

Reply:
Yeah, https://github.com/ARMmbed/mbed-tools-ci-scripts/blob/master/mbed_tools_ci_scripts/assert_news.py#L47 ignores empty line :)

Reply:
Yeah, https://github.com/ARMmbed/mbed-tools-ci-scripts/blob/master/mbed_tools_ci_scripts/assert_news.py#L47 ignores empty line :)